plugins/rollbar/test/decorators/stage_decorator_test.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+require_relative '../test_helper'
+
+SingleCov.covered!
+
+describe Stage do
+  let(:stage) { stages(:test_staging) }
+  let(:attributes) do
+    {
+      webhook_url: "http://rollbar.com/abcdef",
+      access_token: "test",
+      environment: 'staging',
+      stage_id: stage.id
+    }
+  end
+
+  describe "#rollbar" do
```

Comment:
`rollbar` is not a method ... so use `describe "assigning rollbar attributes"` or so